apps/nextjs/src/components/DialogControl/DialogContents.tsx

```diff
@@ -10,8 +10,11 @@ import styled from "styled-components";
 
 import type { DialogTypes } from "../AppComponents/Chat/Chat/types";
 import { useDialog } from "../AppComponents/DialogContext";
+import AdditionalMaterialsError from "./ContentOptions/AdditionalMaterialError";
```

Comment:
import says 'AdditionalMaterialsError' but file name is 'AdditionalMaterialError'